<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-E1NRTU5J" failures="1" tests="2" name="Test(failed)" time="15.571" errors="0" timestamp="2023-12-01T12:34:37 CST">
  <testcase classname="testCases.TC_002_LoginTest" name="verify_Login" time="1.796"/>
  <testcase classname="testCases.TC_001_AccountRegistrationTest" name="verify_Account_Registration" time="1.965">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_001_AccountRegistrationTest.verify_Account_Registration(TC_001_AccountRegistrationTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_Account_Registration -->
</testsuite> <!-- Test(failed) -->
